Source/WebCore/dom/EventPath.cpp

```diff
@@ -177,6 +177,17 @@ void EventPath::setRelatedTarget(Node& origin, Node& relatedNode)
     }
 }
 
+void EventPath::adjustForDisabledFormControl()
+{
+    for (unsigned i = 0; i < m_path.size(); ++i) {
```

Comment:
Where in the spec does it say to do this?

Reply:
As far as I know it doesn't currently but this is Firefox behavior. This is also what Blink did more recently to align:
- https://chromium-review.googlesource.com/c/chromium/src/+/3929060

This is what's being discussed here (seemed to be the most popular behavior):
- https://github.com/whatwg/html/issues/5886

With us doing this too, all 3 engines should agree on this behavior now.

Reply:
I added this information to the commit log.